include/WCSimDetectorConstruction.hh

```diff
@@ -176,6 +179,9 @@ public:
       WCDetCentre[1] = y;
       WCDetCentre[2] = z;
   }
+  
+  // BGO
+  static G4Material* BGO;
```

Comment:
```suggestion
  G4Material* BGO;
```
Can't see the benefit of making this static (we always only have one instance of WCSimDetectorConstruction)